.env

```
@@ -248,7 +248,7 @@ VORTEX_NOTIFY_EMAIL_FROM=webmaster@your-site-domain.example
 # Multiple names can be specified as a comma-separated list of email addresses
 # with optional names in the format "email|name".
 # Example: "to1@example.com|Jane Doe, to2@example.com|John Doe"
-VORTEX_NOTIFY_EMAIL_RECIPIENTS=webmaster@your-site-domain.example
+VORTEX_NOTIFY_EMAIL_RECIPIENTS="webmaster@your-site-domain.example|Webmaster"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)